internal/controllers/machine/machine_controller.go

```diff
@@ -378,13 +379,19 @@ func (r *Reconciler) reconcileDelete(ctx context.Context, cluster *clusterv1.Clu
 				return ctrl.Result{}, err
 			}
 
-			// The DrainingSucceededCondition never exists before the node is drained for the first time,
-			// so its transition time can be used to record the first time draining.
+			// The DrainingSucceededCondition never exists before the node is drained for the first time.
 			// This `if` condition prevents the transition time to be changed more than once.
```

Comment:
Let's delete this line. It's a) not true at the moment b) not relevant anymore

Same for the other occurence below